Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -85,7 +85,7 @@ void DSPEmitter::ClearIRAMandDSPJITCodespaceReset()
   CompileDispatcher();
   m_stub_entry_point = CompileStub();
 
-  for (int i = 0x0000; i < 0x10000; i++)
+  for (size_t i = 0; i < MAX_BLOCKS; i++)
```

Comment:
> Wouldn't this be a nice place to take advantage of std::array by using .size() instead?

I didn't use `.size()` because the iteration applies to more than just that array.

> Also, it looks a little odd that ...

Yeah, I noticed that as well, 

> Would there be an advantage (size, performance or whatever) in making them std::array too?

The only difference is no contents will be allocated on the heap (if the emitter itself isn't allocated on the heap), and will more than double the size of the class (1051400 bytes to 2230976 bytes). It'll also generally be one less pointer access for accessing any container elements ([example](https://godbolt.org/g/MeXtwH)).

I didn't know if it was done like that for a particular reason, so I just left the other containers as is.